lib/configurations.js

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+// load config defaults
+require('./config');
+
+const instances = require('./instances');
+// FIXME: use `Collection` instead
```

Comment:
How instead of what?

Reply:
`walletClient.js` needs to be removed. It used to have the same stuff in it that `Collection` does now. We can't remove `walletClient.js` yet though -- as it has a couple of lines of code that need to be added to account creation that needs to get imported into this module from elsewhere. Once that's done, we can delete `walletClient.js`. This `configurations` stuff needs to be updated to use `Collection` in the meantime.

Reply:
@dlongley are you already doing this or do we need an issue?

Reply:
Already have it marked on a punchlist.